pages/common/gh-issue.md

```diff
@@ -0,0 +1,28 @@
+# gh issue
+
+> Manage GitHub issues from the command line.
+> More information: <https://cli.github.com/manual/gh_issue>.
+
+- Print out the issue:
+
+`gh issue view {{issue_number}}`
+
+- Create a new issue in the web browser:
+
+`gh issue create --web`
+
+- List the last 10 issues with the `bug` label:
+
+`gh issue list --limit {{10}} --label "{{bug}}"`
+
+- List closed issues made by a specific user:
+
+`gh issue list --state closed --author {{username}}`
+
+- Display status of issues relevant to you, in a specific repository:
```

Comment:
We do have a convention surrounding the use of the 2nd person though (e.g. `you`, `your`), in that we avoid it if at all possible. What about this?
```suggestion
- Display the status of relevant issues in a specific repository:
```

Reply:
Umm, this command specifically shows *your* issues, so I would change that to at least "relevant to user" - is that ok?

Reply:
Makes sense to me. 👍🏻